en/services/mission.md

```diff
@@ -258,6 +258,11 @@ The recommended timeout values before resending, and the number of retries are:
 - Retries (max): 5
 
 
+### Invalid Mission Items
+
+TBD <!-- how should systems handle invalid items - ie not supported at all, or on vehicle type ? -->
```

Comment:
@julianoes ArduPilot validates some things on mission upload, but does this in a vehicle-agnostic way, so you might end up with plane mission items in your copter. The solution there is to allow the upload but silently drop mission items that are not understood during execution.

I am imagine PX4 will have same/worse problems, because unlike ArduPilot we might not even clear a mission when new firmware is uploaded (?). Do you know what PX4 does?
- Does it dump items it doesn't understand at upload or execution time?
- Are they validated per-vehicle type on upload or for all vehicles
- What about things like invalid jumps - does PX4 check for validity on upload (ie that a jump isn't outside of index?)